tests/modules/local/maptotsv/main.nf.test

```diff
@@ -0,0 +1,289 @@
+nextflow_process {
+
+    name "Test Process MAP_TO_TSV"
+    script "modules/local/map_to_tsv.nf"
+    process "MAP_TO_TSV"
+
+    test("Single row, basic input") {
+
+        when {
+            params {
+                outdir = "tests/results"
+            }
+            process {
+                """
+                input[0] = new Tuple("sample", "outbreak", "organism", "subtype", "country", "serovar", "age", "date", "source", "special")
+                input[1] = new Tuple(["S1", "1", "Escherichia coli", "EHEC/STEC", "Canada", "O157:H7", "21", "2024/05/30", "beef", "true"])
+                """
+            }
+        }
+
+        then {
+            assert process.success
+
+            with(process.out) {
+                // check outputs
+                assert path(tsv_path[0]).exists()
+                assert nonempty_column_headers == [["sample", "outbreak", "organism", "subtype", "country", "serovar", "age", "date", "source", "special"]]
+
+                // parse tsv file
+                def tsv_text = path(tsv_path[0]).readLines()
```

Comment:
Can you add two additional assert statements for each test:

1. Assert that the appropriate header lines are contained in the output file.
2. Assert that the number of lines is correct (in this case, should be 2 lines).

Reply:
Added in 0ebd7271c224d509091af468c63bf5313067bcb5